tests/test_portchannel.py

```diff
@@ -89,6 +90,69 @@ def test_Portchannel(self, dvs, testlog):
         lagms = lagmtbl.getKeys()
         assert len(lagms) == 0
 
+    def test_Portchannel_lacpkey(self, dvs, testlog):
+        portchannelNamesAuto = [("PortChannel001", "Ethernet0", 1001),
+                            ("PortChannel002", "Ethernet4", 1002),
+                            ("PortChannel2", "Ethernet8", 12),
+                            ("PortChannel000", "Ethernet12", 1000)]
+
+        portchannelNames = [("PortChannel0003", "Ethernet16", 0),
+                            ("PortChannel0004", "Ethernet20", 0),
+                            ("PortChannel0005", "Ethernet24", 564)]
+
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+        # Create PortChannels
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL")
+        fvs = swsscommon.FieldValuePairs(
+            [("admin_status", "up"), ("mtu", "9100"), ("oper_status", "up"), ("lacp_key", "auto")])
+
+        for portchannel in portchannelNamesAuto:
+            tbl.set(portchannel[0], fvs)
+            time.sleep(1)
+
+        fvs_no_lacp_key = swsscommon.FieldValuePairs(
+            [("admin_status", "up"), ("mtu", "9100"), ("oper_status", "up")])
+        tbl.set(portchannelNames[0][0], fvs_no_lacp_key)
+        time.sleep(1)
```

Comment:
why do you have all these sleeps?

Reply:
I had them just because I based my test on the second test in this file that had them, but I removed them and it works